tests/run/unicode_identifiers.pyx

```diff
@@ -184,6 +195,32 @@ class NormalClassΓΓ(Γναμε2):
             pass
         return nestεd
 
+def unicode_kwarg(*,αrg):
```

Comment:
Space.
Also, I think an example where the difference is easier to spot (in different fonts) than for `arg` versus `αrg` would be nice.